README.rst

```diff
@@ -143,19 +143,19 @@ Below is an example usage of ``nise`` for AZURE data::
 
     nise --start-date 2019-08-01 --azure
 
-    nise --start-date 2019-08-01 --azure --azure-container-name container --azure-report-name cur
+    nise --start-date 2019-08-01 --azure --azure-storage-name container --azure-report-name cur
```

Comment:
In the PR I merged (#151) this field was changed to `--azure-container-name`

Reply:
👍 need to re-install after pulling master